embeddings/pipeline/lightning_sequence_labeling.py

```diff
@@ -64,17 +61,20 @@ def __init__(
         self.task_model_kwargs = task_model_kwargs = initialize_kwargs(
             self.DEFAULT_TASK_MODEL_KWARGS, task_model_kwargs
         )
-        self.early_stopping_kwargs = initialize_kwargs(
-            self.DEFAULT_EARLY_STOPPING_KWARGS, early_stopping_kwargs
-        )
         self.task_model_kwargs.update(
             {"train_batch_size": train_batch_size, "eval_batch_size": eval_batch_size}
         )
+        self.early_stopping_kwargs = initialize_kwargs(
+            self.DEFAULT_EARLY_STOPPING_KWARGS, early_stopping_kwargs
+        )
+        self._logging_kwargs = initialize_kwargs(self.DEFAULT_LOGGING_KWARGS, logging_kwargs)
         tokenizer_name_or_path = (
             tokenizer_name_or_path if tokenizer_name_or_path else embedding_name_or_path
         )
 
         output_path = Path(output_path)
+        self.evaluation_filename = evaluation_filename
```

Comment:
`evaluation_filename` is assigned as an instance attribute and then in line 103 we use it as a variable; is it necessary for some attributes saving or is it redundant?

Reply:
same goes to `lightning classification` pipeline